src/main/java/edu/cornell/kfs/pdp/CUPdpParameterConstants.java

```diff
@@ -4,7 +4,18 @@
 
     public static final String WARNING_INACTIVE_VENDOR_TO_EMAIL_ADDRESSES = "WARNING_INACTIVE_VENDOR_TO_EMAIL_ADDRESSES";
 
+    public static final String ACH_PERSONAL_CHECKING_TRANSACTION_CODE = "ACH_PERSONAL_CHECKING_TRANSACTION_CODE";
+    public static final String ACH_PERSONAL_SAVINGS_TRANSACTION_CODE = "ACH_PERSONAL_SAVINGS_TRANSACTION_CODE";
+    public static final String ACH_DIRECT_DEPOSIT_TRANSACTION_TYPE = "ACH_DIRECT_DEPOSIT_TRANSACTION_TYPE";
+    public static final String ACH_SECURE_FIELDS_ALLOWED_IN_ACCOUNT_EMAILS = "ACH_SECURE_FIELDS_ALLOWED_IN_ACCOUNT_EMAILS";

```

Comment:
Yes, the bank name is the only masked field that users want included in the generated emails, and I do not anticipate users needing any other masked ones in there. We could hard-code a bank name property constant if necessary, if we're fine with adjusting it accordingly if the property name ends up changing with the upgrade.